test/e2e/play_kube_test.go

```diff
@@ -6310,4 +6310,85 @@ spec:
 		session.WaitWithDefaultTimeout()
 		Expect(session).Should(ExitWithError(125, "invalid signal: noSuchSignal"))
 	})
+
+	It("test with custom log path from containers.conf", func() {
+		customLogPath := filepath.Join(podmanTest.TempDir, "custom-logs")
+		expectedMessage := "Pod started, checking logs from test"
+
+		conffile := filepath.Join(podmanTest.TempDir, "containers.conf")
+		configContent := fmt.Sprintf(`[containers]
+log_driver = "k8s-file"
+log_path = "%s"
+`, customLogPath)
+
+		err := os.WriteFile(conffile, []byte(configContent), 0644)
+		Expect(err).ToNot(HaveOccurred())
+
+		err = os.MkdirAll(customLogPath, 0755)
+		Expect(err).ToNot(HaveOccurred())
+
+		oldContainersConf := os.Getenv("CONTAINERS_CONF")
+		os.Setenv("CONTAINERS_CONF", conffile)
+		defer os.Setenv("CONTAINERS_CONF", oldContainersConf)
```

Comment:
you don't need to make the get/defer set again dance. All tests unset CONTAINERS_CONF_OVERRIDE/CONTAINERS_CONF. So just do the setenv call to your value.